src/core/Sidebar/SidebarButton.tsx

```diff
@@ -0,0 +1,65 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import cx from 'classnames';
+import React from 'react';
+
+import { useTheme } from '../utils/hooks/useTheme';
+import '@itwin/itwinui-css/css/side-navigation.css';
+import { Button, ButtonProps } from '../Buttons';
+import { Tooltip } from '../Tooltip';
+
+export type SidebarButtonProps = {
+  /**
+   * Whether the sidebar button is active.
+   */
+  isActive?: boolean;
+  /**
+   * If true, button label will also be shown as a tooltip.
+   * By default, tooltip is shown when sidebar is collapsed, and hidden when expanded.
```

Comment:
This comment is weird...Default value is true or false? Or is it some 3rd option.

Reply:
I guess technically the default is undefined. But practically, it is set to true in `Sidebar` when collapsed and false when expanded

Reply:
What about using `title` prop instead of this flag?

Reply:
Removed `showTooltip` and added `title`

Reply:
Removed this prop and moved tooltip logic to `Sidebar`

Reply:
I still think we should use `title` for tooltip content as it allows user to customize tooltip.

Reply:
What do you mean to customize? There we should show tooltip, not title by design.

Reply:
I mean to use `title` value for `Tooltip` content - not to use `children` but `title`.

Reply:
That could be confusing and unintuitive. Plus we'd have to hide the native tooltip by unassigning `title` again 🤔